examples/undocumented/libshogun/labels_binary_fit_sigmoid.cpp

```diff
@@ -6,22 +6,28 @@
 
 #include <shogun/base/init.h>
 #include <shogun/labels/BinaryLabels.h>
+#include <shogun/evaluation/SigmoidCalibration.h>
 
 using namespace shogun;
 
 void test_sigmoid_fitting()
 {
 	CBinaryLabels* labels=new CBinaryLabels(10);
+	CBinaryLabels* predictions=new CBinaryLabels(10);
 	labels->set_values(SGVector<float64_t>(labels->get_num_labels()));
 
 	for (index_t i=0; i<labels->get_num_labels(); ++i)
-		labels->set_value(i%2==0 ? 1 : -1, i);
-
-	labels->get_values().display_vector("scores");
-	labels->scores_to_probabilities();
-	labels->get_values().display_vector("probabilities");
-
-
+	{
+		predictions->set_value(i%2==0 ? 1 : -1, i);
+		labels->set_value(i%4==0 ? 1: -1, i);
+	}
+	predictions->get_values().display_vector("scores");
+	CSigmoidCalibration* sigmoid_calibration = new CSigmoidCalibration();
+	sigmoid_calibration->fit_binary(predictions, labels);
+	CBinaryLabels* calibrated_labels = sigmoid_calibration->calibrate_binary(predictions);
+	calibrated_labels->get_values().display_vector("probabilities");
```

Comment:
why display?

Reply:
I'm not sure myself. It was in the original code https://github.com/shogun-toolbox/shogun/blob/297f9a0c5d4771154515fef2fac865fdc1a1af5e/examples/undocumented/libshogun/labels_binary_fit_sigmoid.cpp#L22

Reply:
I see. Not your fault of course. Would still be better removed :)
In fact, what would be best was to remove the example altogether (another PR) and have a meta example for that